test/Frontend.hs

```diff
@@ -92,6 +95,11 @@ removeNoComment =
 -- Parse Many at once
 --------------------------------------------------------------------------------
 
+parsedIdString :: Either String [Juvix.Frontend.Types.TopLevel]
+parsedIdString =
+  Parser.parse
+    (encodeUtf32BE $ unsafePerformIO . readFile $ "/test/Id-Strings.jvx")
```

Comment:
Hmm, we should find a way to write these tests that does not use `unsafePerformIO` - Tasty can do IO.